### PR TITLE
Reviewer feedback deadline

### PR DESCRIPTION
**Pull request by littledan**
Reviewers should provide feedback a month before the meeting where
the proposal is up for Stage 3 review.

### CODE REVIEW

**Comment by bterlson** [comment]
Yes please! Not sure what the process is to make changes to the process document. Where is the meta-process document??

File: index.html [hunk 1]
```diff
@@ -160,6 +160,8 @@
 
 <p>Anyone can be a reviewer and submit feedback on an in-process addition. The committee should identify designated reviewers for acceptance during the “draft” (stage 2) maturity stage. These reviewers must give their sign-off before a proposal enters the “candidate” (stage 3) maturity stage. Designated reviewers should not be authors of the spec text for the addition and should have expertise applicable to the subject matter. Designated reviewers must be chosen by the committee, not by the proposal's champion.
 
+<p>When reviewers are designated, a target meeting for Stage 3 should be identified. Initial reviewer feedback should be given to the champions one month before that meeting to allow for a back-and-forth ahead of the meeting. The target Stage 3 meeting may be delayed by a champion outside of the meeting at a later time if it is not ready.
```
Comment (ljharb) [inline index.html]: Should it also be made explicit that if the champion fails to provide a Stage 3 candidate that addresses all outstanding feedback by "one month prior", that the proposal might not advance?

Comment (littledan) [inline index.html]: I think this is already described in the document. Reviewer and editor signoff is required.

**Comment by littledan** [comment]
Weren't we going to have a process discussion at some point? Maybe you should just open with this process discussion tomorrow morning.

**Comment by littledan** [comment]
Changed to two weeks. Ready to pull?

**Comment by littledan** [comment]
We discussed this process change at TC39 and agreed that reviews should be in two weeks in advance. There's no enforcement mechanism, this is still moral suasion and you still need consensus. We considered some other process changes, e.g., putting things into the agenda further ahead, but decided to just make this one.

**Comment by bterlson** [comment]
LGTM. What a big improvement for such a @littledan!

**Pull request merged**